paddle/go/pserver/service.go

```diff
@@ -0,0 +1,179 @@
+package pserver
+
+import (
+	"errors"
+	"fmt"
+	"sync"
+)
+
+// ElementType is the type of elements of a Parameter.
+type ElementType int
+
+var ErrUnintialized = errors.New("pserver not initialized")
```

Comment:
After some discussion with @helinwang , we found out that `errors.New()` created objects are of struct type, which can not be a const type (https://golang.org/src/errors/errors.go). Const types are https://golang.org/ref/spec#Constants. There are hacky methods to turn errors to const types https://dave.cheney.net/2016/04/07/constant-errors but we would not encourage.

Generally, use `var ErrorType = errors.New()` is the best. Utill Go supports "const error" types